src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
 import './MoviesList.scss';
+import React from 'react';
 import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, query }) => {
+  const lowerQuery = query.toLowerCase();
+  const visibleMovies = movies
+    .filter(movie => movie.title.toLowerCase().includes(lowerQuery)
+    || movie.description.toLowerCase().includes(lowerQuery));
```

Comment:
Try to keep DRY(don't repeat yourself) pattern, I suggest to do smth like that:

```suggestion
  const isQueryMatch = text => text.toLowerCase().includes(query.toLowerCase())

  const visibleMovies = movies
    .filter(({ title, description })=> isQueryMatch(title) || isQueryMatch(description))
```